### ISSUE TITLE
Reference to undefined variable in `organize.js`

### ISSUE THREAD

**Issue by Alamantus** [Alamantus/GooglePhotosExportOrganizer#6]
Received an email from the website form from 'arthur' with this screenshot:

![JavaScript error message](https://i.imgur.com/Ye23w3z.png)

```
Uncaught Exception:
TypeError: Cannot read property 'formatted' of undefined
  at C:\Users\Eduard\Downloads\GooglePhotosExportOrganizer_windows\Google Photos Export Organizer 1.1.0ads\resources\app.asar\build\organize.js: 124:53
  at FSReqCallback.oncomplete (fs.js: 160:23)
```

**Comment by StephenCYang** [comment]
Is this in the most current release?

**Comment by alymenbr** [comment]
I also came across this issue running the MacOS version. Would you be able to generate a new release containing this fix?

**Comment by Alamantus** [comment]
Unfortunately I haven't had a chance to run the build process yet--getting all 3 builds requires 3 separate computers for me at present.

I will try to get the new release with the fix up by next week!

**Comment by Alamantus** [comment]
This error should be fixed in the new [version 1.1.1 release](https://github.com/Alamantus/GooglePhotosExportOrganizer/releases/tag/v1.1.1). Please let me know if you still keep getting the error!
